CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.12.3 (2019-05-22)
+
+This is a patch release.
+
+### Fixes
+
+- Show tag image for workload in list-images
```

Comment:
This is the title of the PR but it's not self-explanatory. I'd suggest "Show untagged images in fluxctl list-images"

Reply:
Done